packages/api/src/hooks/useLandingCompany.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import useFetch from '../useFetch';
+import useSettings from './useSettings';
+
+/** A custom hook that returns the available landing companies of the user's country. */
+const useLandingCompany = () => {
+    const { data: settings_data } = useSettings();
+    const { data, ...rest } = useFetch('landing_company', {
+        payload: { landing_company: settings_data?.country_code || '' },
+        options: { enabled: Boolean(settings_data?.country_code) },
+    });
+
+    const modified_data = useMemo(() => {
+        if (!data?.landing_company) return;
+        const { financial_company, gaming_company, virtual_company } = data.landing_company;
```

Comment:
@farhan-nurzi-deriv  There are more data inside `data.landing_company` but here we are ignoring them, Why? 🤔

Reply:
because currently we only need the shortcodes, but i will change the code and return all the data 🙏